independent-publisher-connectors/MailJet/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key / Login",
+          "description": "The API Key/Login issued by MailJet",
+          "tooltip": "Provide the API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Secret",
+          "description": "The API Secret/PassWord issued by MailJet",
+          "tooltip": "Provide the API Secret",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@carfup You have agreed that your connector will not have "#007ee5" as iconBrandColor for your connector. Al the IP Connectors must be [set iconBranfColor to "#da3b01"](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).

![image](https://user-images.githubusercontent.com/56276394/148867974-89aeb3d6-f1f6-4da3-be5d-3585816911ba.png)

Reply:
My mistake here. I guess I missed a commit on my end regarding the color...